src/Workspace/index.jsx

```diff
@@ -734,19 +735,7 @@ class Workspace extends React.Component {
         </div>
       );
     } if (this.state.defaultWorkspace && this.state.connectedStatus) {
```

Comment:
there is no more `defaultWorkspace`, so you should change the name of this state variable to better represent the current logic, otherwise it is super confusing to ppl who reads this code